src/components/Modals/CreateSingleItemModal/utils.ts

```diff
@@ -28,3 +31,35 @@ export function validateEnum(name: keyof ItemAssetJson, assetJson: ItemAssetJson
     throw new InvalidEnumValue(value!, values, name)
   }
 }
+
+export function toWearableWithBlobs(file: File, isEmote = false): WearableWithBlobs {
```

Comment:
I _believe_ it can be used for wearables as well, it can turn any model to a `WearableWithBlobs` type. I picked the function from here: https://github.com/decentraland/ui/blob/master/src/components/WearablePreview/WearablePreview.stories.tsx#L43.